plugin_runner/sandbox.py

```diff
@@ -936,6 +938,22 @@ def _safe_import(
 
         return imported_module
 
+    def _safe_sum(self, iterable: Iterable[Any], start: Any = 0) -> Any:
```

Comment:
`_safe_sum` can also be at the root since it does not use any class methods/attributes